frontend/src/entities/configuration/dynamic-configuration.tsx

```diff
@@ -259,11 +267,16 @@ export function DynamicConfiguration () {
                             iconAlt='Update dynamic configuration'
                             variant='primary'
                             onClick={() => {
-                                setModalVisible(true);
+                                if (!isValid) {
+                                    setState({ validateAll: true, formSubmitting: false });
+                                    scrollToInvalid();
+                                } else {
+                                    setModalVisible(true);
+                                }
                             }}
                             loading={state.formSubmitting}
                             data-cy='dynamic-configuration-submit'
-                            disabled={!isValid || state.formSubmitting}
+                            disabled={state.formSubmitting}
```

Comment:
Any reason you removed the `isValid` check here? Looking at your gif it looks like when the text field is blank, the "Save Changes" button is active.

Reply:
thats how we do it in all other form submits (ex: notebook create). If I don't do it this way, then the user would not know why the form is not valid unless we have some sort of useEffect running with setting ValidateAll. We need that validateAll call to trigger the "error" state of the formField.

Anyways, I prefer to keep it similar to what we do with all other forms unless there is a specific reason to have it different here

Reply:
Fair enough, this is how we do it in many places